.github/workflows/go.yml

```diff
@@ -48,3 +48,71 @@ jobs:
 
     - name: Test
       run: go test -v ./...
+
+  build-linux:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: ^1.21
+
+      - name: Get dependencies
+        run: go get -v ./...
+
+      - name: Build for Linux
+        run: |
+          PREFIX="build"
+          OS="linux"
+          APPS="datasetArchiver datasetGetProposal datasetRetriever datasetIngestor datasetCleaner"
+          
+          for GOOS in $OS; do
```

Comment:
this loop is not looping

Reply:
how do you know? what's wrong?

Reply:
OS is single valued so, this is looping on a single value. What's the point?